reputation/microeconomic_reputation.tex

```diff
@@ -0,0 +1,367 @@
+I.e. $V_A(a) = X$ means "when agent $A$ takes part in the agreement $a$, their hapiness changes as if they were given $X$ money".}, i.e.:
+    \begin{itemize}
+        \item For the provider it is the utility of the money received decreased by the utility cost of the hardware/electricity/etc.
+        \item For the requestor it is the utility of the resources obtained decreased by the utility cost of the money spent
+    \end{itemize}
+\item $V_P(a)$/$V_R(a)$ are agreement values from the POV of (respectively) provider/requestor.
+\item $V_{PN}(a)$/$V_{RN}(a)$ are nominal (i.e. negotiated) values of the agreement\footnote{
+   Note that this doesn't translate directly to the agreement amount. E.g. when agreed amount is positive but so low it doesn't cover provider's costs the $V_{PN}$ is negative.
+}. They equal $V_P(a)$/$V_R(a)$ if neither side breached the agreement $a$.
+\item $V_{AL}(a)$ is the value lost by agent $A$ because of the other side breaching the agreement $a$.
+\item $V_{AG}(a)$ is the value gained by agent $A$ when they break the agreement $a$.
+\item $V_{PL}$, $V_{RL}$, $V_{PG}$, $V_{RG}$ are $V_{AL}$/$V_{AG}$ from the POV of provider/requestor.
+\item $C_A$ is the cost of the participation in the Golem market for agent $A$ that is not related to any particular agreement 
+(e.g. the opportunity cost\footnote{\href{https://en.wikipedia.org/wiki/Opportunity\_cost}{https://en.wikipedia.org/wiki/Opportunity\_cost}} of the hardware offered on the market, 
+or the cost of writing the requestor agent).
+\end{itemize}
+
+\subsection{Dissolving the Golem Value}
+
+Utilizing the above notation, we can rewrite the $V_G$ equation as:
+
+\begin{equation}
+    V_G = \sum_{a \in all\_agreements}(V_P(a) + V_R(a)) - \sum_{A \in all\_agents\_using\_Golem}C_A
+\end{equation}
+
+The following equation and it's counterpart from the requestor POV are true by definition:
+
+\begin{equation}
+    V_P(a) = V_{PN}(a) + V_{PG}(a) - V_{PL}(a)
+\end{equation}
+
+By placing them in the previous equation, we get:
+
+\begin{equation}
+\begin{split}
+    V_G = \sum_{a \in all\_agreements}(V_{PN}(a) + V_{PG}(a) - V_{PL}(a) + V_{RN}(a) + V_{RG}(a) - V_{RL}(a)) \\
+            - \sum_{A \in all\_agents\_using\_Golem}C_A \\
+\end{split}
+\end{equation}
+\begin{equation}
+\begin{split}
+    V_G = \sum_{a \in all\_agreements}(V_{PN}(a) + V_{PG}(a)) \\
+          - \sum_{a \in all\_agreements}(V_{PL}(a)- V_{RG}(a)) \\
+          - \sum_{a \in all\_agreements}(V_{RL}(a)- V_{PG}(a)) \\
+          - \sum_{A \in all\_agents\_using\_Golem}C_A
+\end{split}
+\end{equation}
+
+Important note here is that both $\sum_{a \in all\_agreements}(V_{PL}(a)- V_{RG}(a))$ and $\sum_{a \in all\_agreements}(V_{RL}(a)- V_{PG}(a))$ are positive: 
+when someone breaks the agreement, the harm done to the victim is usually greater then the offenders gain\footnote{
```

Comment:
This might be, but is estimating numerical values here feasible?

Reply:
You mention "additional far-reaching assumptions". Would it then be fair to conclude that it is hard to estimate `V_L` and hence potential improvements to `V_G` from reputation?

Reply:
Again, I don't understand, sorry :(

To estimate any numerical value we need assumptions about utility functions. This is true about `V_G` even without any reputation-related component. So I think:
* if we don't assume anything about the utility functions, there's no numerical value we can calculate
* if we make enough assumptions, they will let us estimate `V_L` using some repuatation-related metrics (I'm not certain about this statement, but I believe it is true)